components/cam/src/dynamics/se/inidat.F90

```diff
@@ -109,6 +118,40 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
       end if
     end if
 
+!   Determine column closest to SCM point
+    if (single_column) then
+      if (scmlon .lt. 0._r8) then
+        scmposlon=scmlon+360._r8
+      else
+        scmposlon=scmlon
+      endif 
+      minpoint=10000.0_r8
+      ie_scm=1
```

Comment:
Would it be better to set `ie_scm`, `i_scm`, `j_scm` and `indx_scm` to 0 and have a check to make sure that this loop actually found a nearest point?  I can't imagine a case where you won't find a point closer than 10000.0_r8 but just in case there is some user error having a check will make sure the code won't default to the first point in the mesh might be good.